recipes/langgraph-checkpoint-aws/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "langgraph-checkpoint-aws" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
```

Comment:
You need to set this only if the package min is higher than conda-forge's min.

```suggestion
```

Reply:
Fixed, thanks!